### ISSUE TITLE
Not able to install using public IPs

### ISSUE THREAD

**Issue by jnikodemus** [wazuh/wazuh-installation-assistant#331]
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 4.9 | Cluster | Install | Ubuntu 24.04 |

I'm trying to install the Wazuh cluster using the assisted installation. Unfortunately, this installation is interrupted because it involves public IPs. Without circumventing the installation script, an installation is not possible/allowed. Another flag such as "ignorePublicIp" could solve this problem. Related Issue wazuh/wazuh-packages#1627

**Comment by fcaffieri** [comment]
Indeed, the Wazuh installation assistant script, for security reasons, has been configured to prevent unexpected exposure of services to the internet. For this reason, it validates that public IPs are not used because all the core components should communicate through a private network for security (not exposing APIs/servers to the Internet) and performance (having low latency when using a private network) reasons. Using a public IP could lead to more significant issues, such as exposing services to the public internet or running with an invalid configuration, leading to downtime or security breaches.
If you want to assume the exposed risks, you can modify the script to skip this validation of the public IPs by commenting out the following code.

```
for ip in "${all_ips[@]}"; do
    isIP=$(echo "${ip}" | grep -P "^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$")
    if [[ -n "${isIP}" ]]; then
        if ! cert_checkPrivateIp "$ip"; then
            common_logger -e "The IP ${ip} is public."
            exit 1
        fi
    fi
done
```

**Comment by c-bordon** [comment]
Closed due to lack of activity
